components/footer/footer.module.scss

```diff
@@ -1,6 +1,6 @@
 .infoRepo {
   font-weight: 100;
-  margin: 0;
+  margin: 10px auto 10px auto;
```

Comment:
This could be reduced more,
top left bottom right
top,bottom left,right

Reply:
I could change left and right margins from auto to 0px and also adjust the top and bottom margins so that the footer issue is fixed. okay?

Reply:
Did you go through the link?
What does it say?
Instead of repeating 10px twice, we can call it once if both are same values.

Reply:
I am not sure about requirements, ask @sumitd94  please

Reply:
yeah i studied the link and okay i will group in the same values together and pull again